homeassistant/components/energyzero/services.py

```diff
@@ -107,23 +131,20 @@ async def __get_prices(
     return __serialize_prices(data)
 
 
-@callback
-def async_register_services(
-    hass: HomeAssistant, coordinator: EnergyZeroDataUpdateCoordinator
-):
+async def async_setup_services(hass: HomeAssistant) -> None:
```

Comment:
This should still be a async callback. We don't await inside.

Reply:
I've used the `fully_kiosk` as inspiration here. If you feel like, it you can also change it there.